luigi/postgres.py

```diff
@@ -25,6 +25,7 @@
 import tempfile
 
 from luigi import six
+from luigi.six.moves import xrange
```

Comment:
or you can just change line 324 from `xrange(2)` to `range(2)`. 2 is not an important value to use xrange in my opinion :)